src/components/Form.js

```diff
@@ -149,12 +150,30 @@ export default class Form extends Component {
     }
   };
 
+  getUsedFormData = event => {
+    const fields = event.target.elements;
+    const inputNames = Object.keys(fields).reduce((filtered, i) => {
+      const fieldName = fields[i].name;
+      if (fieldName) {
+        filtered.push(fieldName);
+      }
+      return filtered;
+    }, []);
+
+    return _pick(this.state.formData, inputNames);
+  };
+
   onSubmit = event => {
     event.preventDefault();
     event.persist();
+    let newFormData = this.state.formData;
+
+    if (this.props.omitUnusedData) {
```

Comment:
```suggestion
    if (this.props.omitUnusedData === true) {
```

Reply:
its defaulted to `false`  in the default props and prop type is set to boolean, so we thought this was reasonable, but can update that no problem.